inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -387,6 +387,8 @@ void InferenceEngine::details::CNNLayerCreator::on_adapter(const std::string& na
     } else if (auto a = ::ngraph::as_type<::ngraph::AttributeAdapter<std::vector<std::shared_ptr<
     ngraph::op::util::SubGraphOp::OutputDescription>>>>(& adapter)) {
         (void)a;
+    } else if (auto a = ::ngraph::as_type<::ngraph::AttributeAdapter<ngraph::op::v5::Loop::SpecialBodyPorts>>(& adapter)) {
```

Comment:
why do we need this empty "else if" branches?

Reply:
We need them because we switched from NodeConverter to visitorAPI when converting to CNNNetwork. They are empty because this convertion is already done in `createSubGraphLayer()` :
```
    addSpecificCreator({"Loop"}, [](const std::shared_ptr<::ngraph::Node>& node, const std::map<std::string, std::string>& params) -> CNNLayerPtr {
        auto res = createSubGraphLayer(node);
        res->type = "Loop";
        return res;
```

Reply:
got it, didn't notice THROW_IE_EXCEPTION in "else" branch